packages/cli/lib/auth.mjs

```diff
@@ -0,0 +1,140 @@
+// @ts-check
+import express from 'express';
+import getPort from 'get-port';
+import nconf from 'nconf';
+import open from 'open';
+import openidClient from 'openid-client';
+import { dirname, join } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { logger } from './utils.mjs';
+
+const BASE_URL = process.env.BASE_URL || process.env.PLATFORM_URL;
+const CLIENT_ID = process.env.CLIENT_ID || process.env.API_USER || 'flow-cli';
+const CLIENT_SECRET = process.env.CLIENT_SECRET || process.env.API_KEY;
+const REALM = process.env.REALM;
+const BUFFER = 120;
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+const viewsPath = join(__dirname, 'views');
+
+let server = null;
+nconf.file({ file: join(__dirname, 'config') });
+
+export async function getAccessToken(baseUrl = BASE_URL, realm = REALM) {
+  checkEnvironment([
+    ['BASE_URL', baseUrl],
+    ['REALM', realm],
+  ]);
+
+  nconf.load();
+  let tokenSet = nconf.get(baseUrl.replaceAll(':', '')) || {};
```

Comment:
if you replace this with `.replace(new RegExp(':', 'g'), '')` we will stay node v14 compatible